src/components/LocationPage/ChartBlock.tsx

```diff
@@ -63,15 +67,19 @@ function ChartBlock(props: {
             projections={projections}
             height={chartHeight}
           />
-          <LocationPageSectionFooter>
-            <DisclaimerWrapper>{disclaimerContent}</DisclaimerWrapper>
-            <ShareButtons
-              chartIdentifier={metric}
-              region={region}
-              stats={stats}
-              showEmbedButton={false}
-            />
-          </LocationPageSectionFooter>
+          <FooterWrapper>
+            <Footer>
+              <DisclaimerWrapper>{disclaimerContent}</DisclaimerWrapper>
+              <ButtonContainer>
```

Comment:
hm this change introduces 2 new wrappers that werent previously needed

Reply:
Yea, the `FooterWrapper` was introduced to add some white space at the top 😬 (see point above in the PR description). I was reluctant to add this wrapper cos I wanted to minimize the number of wrappers. I tried adding this space to the `Footer` component instead but that will cause the vertical spacing of the following section in mobile mode to be off. 

![image](https://user-images.githubusercontent.com/46338131/120374702-1b7f0b00-c2d7-11eb-8a34-35bec11085fc.png)